torch/nn/quantizable/modules/activation.py

```diff
@@ -0,0 +1,368 @@
+        observed.bias_v = other.bias_v
+        observed.qconfig = other.qconfig
+
+        # Set the linear weights
+        observed.out_proj.weight = other.out_proj.weight  # type: ignore
+        observed.out_proj.bias = other.out_proj.bias  # type: ignore
+        if other._qkv_same_embed_dim:
+            # Use separate params
+            bias = other.in_proj_bias
+            _start = 0
+            _end = _start + other.embed_dim
+            weight = other.in_proj_weight[_start:_end, :]
+            if bias is not None:
+                bias = torch.nn.Parameter(bias[_start:_end], bias.requires_grad)
+            observed.linear_Q.weight = torch.nn.Parameter(weight,
+                                                          weight.requires_grad)
+            observed.linear_Q.bias = bias
+
+            bias = other.in_proj_bias
+            _start = _end
+            _end = _start + other.embed_dim
+            weight = other.in_proj_weight[_start:_end, :]
+            if bias is not None:
+                bias = torch.nn.Parameter(bias[_start:_end], bias.requires_grad)
+            observed.linear_K.weight = torch.nn.Parameter(weight,
+                                                          weight.requires_grad)
+            observed.linear_K.bias = bias
+
+            bias = other.in_proj_bias
+            _start = _end
+            weight = other.in_proj_weight[_start:, :]
+            if bias is not None:
+                bias = torch.nn.Parameter(bias[_start:], bias.requires_grad)
+            observed.linear_V.weight = torch.nn.Parameter(weight,
+                                                          weight.requires_grad)
+            observed.linear_V.bias = bias
+        else:
+            observed.linear_Q = other.q_proj_weight
+            observed.linear_K = other.k_proj_weight
+            observed.linear_V = other.v_proj_weight
+            if other.in_proj_bias is None:
+                observed.linear_Q.bias = None  # type: ignore
+                observed.linear_K.bias = None  # type: ignore
+                observed.linear_V.bias = None  # type: ignore
+            else:
+                observed.linear_Q.bias = other.in_proj_bias[0:other.embed_dim]
+                observed.linear_K.bias = other.in_proj_bias[other.embed_dim:(other.embed_dim * 2)]
+                observed.linear_V.bias = other.in_proj_bias[(other.embed_dim * 2):]
+        observed.eval()
+        # Explicit prepare
+        observed = torch.quantization.prepare(observed, inplace=True)
+        return observed
+
+    @classmethod
+    def from_observed(cls, other):
+        converted = torch.quantization.convert(other, mapping=None,
+                                               inplace=False,
+                                               remove_qconfig=True,
+                                               convert_custom_config_dict=None)
+        # Remove the parameters for the bias_k and bias_v to quantize them
```

Comment:
We do not need to quantize the bias, our kernels handle them in fp32

Reply:
bias_k is not a bias for linear -- it's additional column in the key tensor. Same for the bias_v